.appveyor.yml

```diff
@@ -46,7 +41,7 @@ for:
             Invoke-WebRequest $env:CYG_URL -OutFile $env:CYG_CACHED_SETUP
           }
       - cmd: >-
-          %CYG_CACHED_SETUP% --quiet-mode --no-shortcuts --no-startmenu --no-desktop --upgrade-also --only-site --site "%CYG_MIRROR%" --root "%CYG_ROOT%"  --local-package-dir "%CYG_CACHE%" --packages "%CYG_PACKAGES%"
+          %CYG_CACHED_SETUP% --no-verify --quiet-mode --no-shortcuts --no-startmenu --no-desktop --upgrade-also --only-site --site "%CYG_MIRROR%" --root "%CYG_ROOT%"  --local-package-dir "%CYG_CACHE%" --packages "%CYG_PACKAGES%"
```

Comment:
Why do you need the `--no-verify`?

Reply:
The archived versions of Cygwin in the "Time Machine" repo don't keep/retain the various file signature files, so this is required in order to use the older versions.
There is perhaps a chance of a Man-in-the-Middle type attack here if the "Time Machine" repo ever was compromised.
However, since we don't actually use any of the resulting build artifacts from the Cygwin build (ie: we don't publish the generated binaries), and the AppVeyor CI builds only have read-only access to the ICU repo, and since the AppVeyor VMs are reset/wiped on build completion, this seems like it is "okay" for the purpose of build validation.